src/read.c

```diff
@@ -157,6 +157,35 @@ char *buf;
     return erode_obj_text(tshirt, buf);
 }
 
+/* return the style of a hawaiian shirt */
+char *
+aloha_design(shirt, buf)
+struct obj *shirt;
+char *buf;
+{
+    unsigned i;
+
+    static const char *aloha_styles[] = {
+        "flamingos", "parrots", "toucans", "birds of paradise", /* birds */
+        "angelfish", "tropical fish", /* fish */
+        "plumerias", "orchids", "hibiscus flowers", "palm leaves", "palm trees", /* plants */
+        "hula dancers", "sailboats", "surfboards", "ukuleles", /* other */
+    };
+
+    static const char *aloha_bgs[] = {
+        "purple", "yellow", "red", "blue", "white", /* solid colors */
+        "abstract", "geometric", "patterened", "checkered", "naturalistic", /* adjectives */
```

Comment:
"patterned"?

Reply:
Thanks for pointing this out!

Reply:
Quite welcome. (As it happens, I was born in Hawaii...)